content/blog/2022/11/2022-11-17-jenkins-election-candidates.adoc

```diff
@@ -0,0 +1,131 @@
+I would like to make Jenkins a great place to contribute, and to get more beginners and regular contributors involved in the community on a regular basis.
+Onboarding more individual contributors would be one of my key targets, for example, through Jenkins' continued participation in Google Summer of Code.
+
+On the technical aspect, I plan to move automatic plugin and component releases forward, and focus on the support and extension of modern technologies, such as Java 17 and beyond.
+
+Affiliations: Student of business science
+
+link:https://github.com/NotMyFault[GitHub]  
+link:https://twitter.com/NotMyFault_OG[Twitter]
+
+=== Board Member - Ullrich Hafner
+
+I have been an active contributor in the Jenkins project since 2007.
+Currently, I am the maintainer of several Jenkins plugins, mostly in the area of build result visualization using modern JS based user interfaces: e.g., I am the maintainer of the warnings, code coverage and git-forensics plugins as well as several UI related JS wrapper plugins like Bootstrap or ECharts.
+I am a member of the Jenkins UX SIG and I formerly was a member of the Jenkins Governance Board during the 2019/2021 term.
+I am a professor of Software Engineering at the University of Applied Sciences Munich, where I also try to win new Jenkins contributors by letting students develop new features in their student projects and theses.
+I am also helping new contributors (and students) in our Jenkins forums or by giving some talks in our Jenkins Meetups.
+It would be a pleasure for me to serve again as a member of the governance board.
+In the board I can help the Jenkins project to evolve and follow a strict vision, that will focus on the needs of the users. 
+
+Affiliations: University of Applied Sciences Munich
+
+link:https://github.com/uhafner[Github]
+
+== New Officers
+
+=== Release Officer - Tim Jacomb
+
+I have been a user of Jenkins for the last 10 years and a regular contributor since 2018.
+I began with maintaining the Slack plugin and over the last couple of years I have since expanded that to many more plugins and the Jenkins core.
+These are some of the components I maintain when I have time: Slack, Azure Key Vault, Junit, most of the Database plugins, Dark theme, Plugin installation manager, Jenkins Helm chart, and Configuration as code plugin.
+I am also a member of the Jenkins infrastructure team, and I was involved in the release automation project and the mirrors modernisation effort, along with the day to day support helping people regain access to accounts etc.
+As a Release Officer I would like to increase automation, ease onboarding of new contributors to the release team, and ensure that responsibilities rotate among people so that I wouldn't be a bottleneck for any task.
+
+Affiliations: Kainos
+
+link:https://github.com/timja[Github]
+
+=== Events Officer - Alyssa Tong
+
+"It has been a privilege for me to be part of the Jenkins community and serve as Events Officer in 2022.
```

Comment:
```suggestion
It has been a privilege for me to be part of the Jenkins community and serve as Events Officer in 2022.
```

Reply:
@alyssat are you OK with this proposed change to your personal statement?